tests/test_ext_autodoc.py

```diff
@@ -2345,7 +2345,11 @@ def test_autodoc(app, status, warning):
 my_name
 
 alias of Foo"""
-    assert warning.getvalue() == ''
+    warning_lines = [line for line in warning.getvalue().split('\n') if line]
+    # This might be a bug. We're not picking up the TypeVar T.
```

Comment:
To clarify: the test file `tests/roots/test-ext-autodoc/target/typehints_lazy.py` causes this warning with or without my changes. Not sure if I did something wrong in the test example or if this needs fixing in autodoc.